frontend/src/app/main/ui/workspace/context_menu.cljs

```diff
@@ -59,6 +59,12 @@
            on-unmount children is-selected icon disabled value]}]
   (let [submenu-ref (mf/use-ref nil)
         hovering?   (mf/use-ref false)
+        handle-menu-click
+        (fn [on-click]
+          (fn [event]
+            (st/emit! dw/hide-context-menu)
+            (when on-click (on-click event))))
```

Comment:
With that code you are generating a new callback each time you render a component, and this is a bad practice. It has no substantial relevance for this code, but I think we should avoid this kind of patters for avoid future copy paste.

I suggest using something like this:

```clojure
(let [on-click'
      (mf/use-fn
       (mf/deps on-click)
       (fn [event]
         (st/emit! dw/hide-context-menu)
         (when on-click
           (on-click event))))
      ...
```